Plugins/FormatSwift/Plugin.swift

```diff
@@ -53,21 +64,34 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
   ///    `excluded` configuration in our `swiftlint.yml`.
   ///  - We could lint `context.package.targets.map { $0.directory }`, but that excludes
   ///    plugin targets, which include Swift code that we want to lint.
-  private func inputPaths(context: PluginContext) throws -> [String] {
+  private func inputPaths(context: PluginContext, excluding subfoldersToExclude: [String]) throws -> [String] {
     let packageDirectoryContents = try FileManager.default.contentsOfDirectory(
       at: URL(fileURLWithPath: context.package.directory.string),
       includingPropertiesForKeys: nil,
       options: [.skipsHiddenFiles])
 
     let subdirectories = packageDirectoryContents.filter { $0.hasDirectoryPath }
     let rootSwiftFiles = packageDirectoryContents.filter { $0.pathExtension.hasSuffix("swift") }
-    return (subdirectories + rootSwiftFiles).map { $0.path }
+
+    return (subdirectories + rootSwiftFiles)
+      .map { $0.path }
+      // Filter out any directories included in the `--exclude` option
+      .filter { path in
+        for subfoldersToExclude in subfoldersToExclude {
+          if path.hasSuffix(subfoldersToExclude) {
+            return false
+          }
+        }
+
+        return true
+      }
   }
 
 }
 
-// MARK: - LintError
+// MARK: - Error
 
-enum LintError: Error {
+enum Error: Swift.Error {
```

Comment:
nit: feels kinda confusing to name this the same as Swift.Error especially since it isn't nested

Reply:
haha sure I'll rename this to `CommandError` or something